### ISSUE TITLE
TestMergeByTimeProcessor

### ISSUE THREAD

**Issue by bossenti** [apache/streampipes#2772]

**Comment by pambrocio** [comment]
please assign this to me

**Comment by pambrocio** [comment]
build successfull
```
[INFO] 
[INFO] Apache StreamPipes ................................. SUCCESS [  1.994 s]
[INFO] streampipes-archetype-extensions-jvm ............... SUCCESS [  0.811 s]
[INFO] streampipes-archetype-pe-sinks-flink ............... SUCCESS [  0.066 s]
[INFO] streampipes-archetype-pe-processors-flink .......... SUCCESS [  0.083 s]
[INFO] StreamPipes Commons ................................ SUCCESS [  2.708 s]
[INFO] streampipes-model-shared ........................... SUCCESS [  1.054 s]
[INFO] streampipes-vocabulary ............................. SUCCESS [  1.802 s]
[INFO] streampipes-model .................................. SUCCESS [  4.567 s]
[INFO] streampipes-dataformat ............................. SUCCESS [  1.025 s]
[INFO] streampipes-messaging .............................. SUCCESS [  0.880 s]
[INFO] streampipes-client-api ............................. SUCCESS [  1.170 s]
[INFO] streampipes-dataformat-json ........................ SUCCESS [  0.897 s]
[INFO] streampipes-dataformat-cbor ........................ SUCCESS [  0.922 s]
[INFO] streampipes-dataformat-fst ......................... SUCCESS [  0.842 s]
[INFO] streampipes-security-jwt ........................... SUCCESS [  0.963 s]
[INFO] streampipes-model-client ........................... SUCCESS [  0.608 s]
[INFO] streampipes-extensions-api ......................... SUCCESS [  1.572 s]
[INFO] streampipes-test-utils ............................. SUCCESS [  1.079 s]
[INFO] streampipes-serializers-json ....................... SUCCESS [  1.220 s]
[INFO] streampipes-client ................................. SUCCESS [  1.120 s]
[INFO] streampipes-measurement-units ...................... SUCCESS [  1.698 s]
[INFO] streampipes-sdk .................................... SUCCESS [  3.481 s]
[INFO] streampipes-connect-shared ......................... SUCCESS [  2.221 s]
[INFO] streampipes-service-discovery-api .................. SUCCESS [  0.933 s]
[INFO] streampipes-dataformat-smile ....................... SUCCESS [  0.904 s]
[INFO] streampipes-storage-api ............................ SUCCESS [  1.269 s]
[INFO] streampipes-storage-couchdb ........................ SUCCESS [  2.435 s]
[INFO] streampipes-storage-management ..................... SUCCESS [  0.926 s]
[INFO] streampipes-user-management ........................ SUCCESS [  1.740 s]
[INFO] streampipes-mail ................................... SUCCESS [  1.358 s]
[INFO] streampipes-resource-management .................... SUCCESS [  2.352 s]
[INFO] streampipes-service-discovery ...................... SUCCESS [  0.864 s]
[INFO] streampipes-pipeline-management .................... SUCCESS [  3.735 s]
[INFO] streampipes-connect-management ..................... SUCCESS [  2.945 s]
[INFO] streampipes-extensions-management .................. SUCCESS [  3.275 s]
[INFO] streampipes-data-explorer-commons .................. SUCCESS [  3.030 s]
[INFO] streampipes-data-explorer .......................... SUCCESS [  2.258 s]
[INFO] streampipes-data-export ............................ SUCCESS [  1.746 s]
[INFO] streampipes-service-base ........................... SUCCESS [  1.190 s]
[INFO] streampipes-rest-shared ............................ SUCCESS [  1.181 s]
[INFO] streampipes-rest-extensions ........................ SUCCESS [  2.301 s]
[INFO] streampipes-service-extensions ..................... SUCCESS [  1.698 s]
[INFO] Apache StreamPipes Maven Plugin .................... SUCCESS [  4.298 s]
[INFO] streampipes-extensions ............................. SUCCESS [  0.008 s]
[INFO] streampipes-connectors-mqtt ........................ SUCCESS [  1.320 s]
[INFO] streampipes-pipeline-elements-shared ............... SUCCESS [  0.828 s]
[INFO] streampipes-connect-adapters ....................... SUCCESS [  0.901 s]
[INFO] streampipes-connect-adapters-iiot .................. SUCCESS [  2.443 s]
[INFO] streampipes-wrapper ................................ SUCCESS [  1.514 s]
[INFO] StreamPipes Wrapper for Standalone Pipeline Element Implementations SUCCESS [  1.625 s]
[INFO] streampipes-connectors-influx ...................... SUCCESS [  1.483 s]
[INFO] streampipes-messaging-kafka ........................ SUCCESS [  1.115 s]
[INFO] streampipes-connectors-kafka ....................... SUCCESS [  1.344 s]
[INFO] streampipes-messaging-nats ......................... SUCCESS [  0.903 s]
[INFO] streampipes-connectors-nats ........................ SUCCESS [  1.087 s]
[INFO] streampipes-connectors-opcua ....................... SUCCESS [  2.867 s]
[INFO] streampipes-connectors-pulsar ...................... SUCCESS [  2.816 s]
[INFO] streampipes-connectors-rocketmq .................... SUCCESS [  2.536 s]
[INFO] streampipes-connectors-tubemq ...................... SUCCESS [  1.186 s]
[INFO] streampipes-processors-filters-jvm ................. SUCCESS [  3.611 s]
[INFO] streampipes-connectors-plc ......................... SUCCESS [  2.190 s]
[INFO] streampipes-messaging-mqtt ......................... SUCCESS [  1.122 s]
[INFO] streampipes-messaging-pulsar ....................... SUCCESS [  1.159 s]
[INFO] streampipes-wrapper-siddhi ......................... SUCCESS [  2.077 s]
[INFO] streampipes-processors-filters-siddhi .............. SUCCESS [  2.025 s]
[INFO] streampipes-processors-transformation-jvm .......... SUCCESS [  4.361 s]
[INFO] streampipes-messaging-jms .......................... SUCCESS [  0.930 s]
[INFO] streampipes-sinks-brokers-jvm ...................... SUCCESS [  1.388 s]
[INFO] streampipes-sinks-databases-jvm .................... SUCCESS [  1.784 s]
[INFO] streampipes-sinks-internal-jvm ..................... SUCCESS [  2.218 s]
[INFO] streampipes-sinks-notifications-jvm ................ SUCCESS [  2.766 s]
[INFO] streampipes-processors-enricher-jvm ................ SUCCESS [  1.740 s]
[INFO] streampipes-processors-change-detection-jvm ........ SUCCESS [  1.203 s]
[INFO] streampipes-extensions-all-iiot .................... SUCCESS [  2.120 s]
[INFO] streampipes-processors-image-processing-jvm ........ SUCCESS [  1.469 s]
[INFO] streampipes-processors-text-mining-jvm ............. SUCCESS [  1.300 s]
[INFO] streampipes-processors-geo-jvm ..................... SUCCESS [  1.965 s]
[INFO] streampipes-extensions-all-jvm ..................... SUCCESS [  2.902 s]
[INFO] streampipes-extensions-iiot-minimal ................ SUCCESS [  1.745 s]
[INFO] StreamPipes Integration Tests ...................... SUCCESS [ 26.670 s]
[INFO] streampipes-rest-core-base ......................... SUCCESS [  1.035 s]
[INFO] StreamPipes REST API ............................... SUCCESS [  2.420 s]
[INFO] streampipes-platform-services ...................... SUCCESS [  1.805 s]
[INFO] streampipes-sdk-bundle ............................. SUCCESS [  0.206 s]
[INFO] StreamPipes Core ................................... SUCCESS [  2.859 s]
[INFO] streampipes-service-core-minimal ................... SUCCESS [  1.296 s]
[INFO] streampipes-wrapper-distributed .................... SUCCESS [  1.176 s]
[INFO] StreamPipes Wrapper for Apache Flink ............... SUCCESS [  1.832 s]
[INFO] streampipes-wrapper-kafka-streams .................. SUCCESS [  1.274 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:58 min
[INFO] Finished at: 2024-04-25T09:34:29+08:00
[INFO] ------------------------------------------------------------------------
paulo@PRED:~/ASF/streampipes$ git status
On branch 2772-fix-TestMergeByTimeProcessor
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   streampipes-extensions/streampipes-processors-filters-jvm/src/test/java/org/apache/streampipes/processors/filters/jvm/processor/merge/TestMergeByTimeProcessor.java

no changes added to commit (use "git add" and/or "git commit -a")
paulo@PRED:~/ASF/streampipes$ 

```

**Comment by pambrocio** [comment]
PR https://github.com/apache/streampipes/pull/2783

**Comment by tenthe** [comment]
Fixed in PR #2375.
